octodns/provider/azuredns.py

```diff
@@ -0,0 +1,423 @@
+#
+#
+#
+
+from __future__ import absolute_import, division, print_function, \
+    unicode_literals
+
+from azure.common.credentials import ServicePrincipalCredentials
+from azure.mgmt.dns import DnsManagementClient
+from azure.mgmt.dns.models import ARecord, AaaaRecord, CnameRecord, MxRecord, \
+    SrvRecord, NsRecord, PtrRecord, TxtRecord, Zone
```

Comment:
Could you advice on the best way to resolve this? These imports are actually being used implicitly in line 72:  `self.params = self.params(record.data, key_name, eval(class_name))`
where `class_name` is one of the record classes. Does pyflakes support an ignore line comment and if so, would that be acceptable to you to use stylistically?